src/module-registries/requirejs-registry.ts

```diff
@@ -0,0 +1,54 @@
+declare var requirejs;
+declare var require;
+declare var console;
+
+import { ModuleRegistry } from '../module-registry';
+
+import {
+  deserializeSpecifier
+} from '@glimmer/di';
+
+export default class RequireJSRegistry implements ModuleRegistry {
+  private _config: any;
+  private _modulePrefix: string;
+
+  constructor(config: any, modulePrefix: string) {
+    this._config = config;
+    this._modulePrefix = modulePrefix;
+  }
+
+  normalize(specifier: string): string {
+    let s = deserializeSpecifier(specifier);
+
+    let type = s.collection === 'main' ? s.type : s.collection;
+    let collection = this._config.collections[s.collection];
+    let group = collection && collection.group;
+
+    if (group) {
+      type = `${group}/${type}`;
+    }
+
+    let path = `${s.rootName}/${this._modulePrefix}/${type}`;
+
+    if (s.name !== 'main') {
+      path += `/${s.name}`;
+    }
+
+    if (s.collection !== 'main') {
+      path += `/${s.type}`;
+    }
+
+    console.log(`requirejs ${specifier} -> ${path}`);
```

Comment:
Was this for debugging purposes? Or is it stripped at build or something I'm not thinking of.